Yosemite/Yosemite/Base/StoresManager.swift

```diff
@@ -20,7 +20,7 @@ public protocol StoresManager {
     /// Switches the internal state to Authenticated.
     ///
     @discardableResult
-    func authenticate(credentials: Credentials) -> StoresManager
+    func authenticate(credentials: WPCOMCredentials) -> StoresManager
```

Comment:
Can we keep `Credentials` as the input type here? We'll need to support authenticating with WPORG credentials here too.